xrpl/asyncio/account/main.py

```diff
@@ -97,11 +99,32 @@ async def get_account_root(
 
     Returns:
         The AccountRoot dictionary for the address.
+
+    Raises:
+        XRPLRequestFailureException: if the rippled API call fails.
     """
-    account_info = await get_account_info(address, client, ledger_index)
+    classic_address = address
+
+    if is_valid_xaddress(address):
+        classic_address, _, _ = xaddress_to_classic_address(address)
+
```

Comment:
Isn't there a helper method that does this conversion for you?

Reply:
I don't believe so, it seems that other logic in this file does it this way